controller/customdomain_controller.go

```diff
@@ -287,6 +289,14 @@ func (r *CustomDomainReconciler) Reconcile(ctx context.Context, request ctrl.Req
 						},
 					},
 				}
+			} else if isAWS && lbType == "NLB" {
```

Comment:
The first condition already checks `isAWS`, so you can next the checks
```
if isAWS{ 
    if lbType == "NLB" {
    } 
}
```

Reply:
Thanks for noticing that!